packages/test-vitest/vitest.config.ts

```diff
@@ -0,0 +1,12 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    browser: {
+      enabled: true,
+      name: 'chrome', // browser name is required
```

Comment:
I'm using Brave Browser but I think it shows itself as Chrome, I will test it later.